Makefile

```diff
@@ -129,9 +129,11 @@ e2e:
 e2e/operator-registry: ## Run e2e registry tests
 	$(MAKE) e2e WHAT=operator-registry
 
-ARTIFACT_DIR ?= '$(ROOT_DIR)'
 e2e/olm: ## Run e2e olm tests
-	$(MAKE) e2e WHAT=operator-lifecycle-manager E2E_CATALOG_NS=openshift-marketplace E2E_INSTALL_NS=openshift-operator-lifecycle-manager E2E_TEST_NS=openshift-operators E2E_TIMEOUT=135m KUBECTL=oc E2E_GINKGO_OPTS="--junit-report olm-e2e-junit.xml"
+	$(MAKE) e2e WHAT=operator-lifecycle-manager E2E_CATALOG_NS=openshift-marketplace \
+		E2E_INSTALL_NS=openshift-operator-lifecycle-manager E2E_TEST_NS=openshift-operators \
+		E2E_TIMEOUT=135m KUBECTL=oc \
+		E2E_GINKGO_OPTS="$(if $(ARTIFACT_DIR),--output-dir='$(ARTIFACT_DIR)' )--junit-report olm-e2e-junit.xml"
```

Comment:
I think you need a space just before `--junit-report` ?
It would look a _bit_ nicer, and be a _bit_ easier to read, if each variable were on it's own line (right now, it's pairs)

Reply:
Or, is the space after `--output-dir='...'` sufficient?

Reply:
yeah, I was in between putting the space inside the if or after it. Guess it makes more sense for the script to be legible than the eventual command to have empty spaces. I was indexing on the log reading rather than the code reading XD I can fix that.

Reply:
Alternatively, if you have bandwidth, I should have invited you as a collaborator to my fork. If you could push that change through so we don't need to wait for tomorrow that'd be sweeeett

Reply:
Fixed ^^